packages/components/src/components/drawer/drawer.stories.ts

```diff
@@ -43,371 +44,74 @@ export default {
-                    part === 'base' ? '' : 'outline-offset: -2px'
-                  };}</style><div id="part-${part}">%TEMPLATE%</div>`
-                }
-              ]
-            }
-          },
-          constants: [
-            {
-              type: 'template',
-              name: 'width',
-              value: `
-                <div style="width: 600px; height: 600px; position: relative;">%TEMPLATE%
-                </div>
-              `
-            }
-          ],
-          args
-        })
-      )}
-    `;
-  }
-};
-
-export const Mouseless = {
-  parameters: { controls: { exclude: ['default', 'header', 'footer', 'contained', 'no-header'] } },
-  render: (args: any) => {
-    return html`<div class="mouseless" style="width: auto; height: 95vh; position: relative;">
-      ${generateTemplate({
-        args,
-        constants: [
-          {
-            type: 'slot',
-            name: 'default',
-            value: `<input autofocus placeholder="Placeholder" class="mt-3 block w-full rounded-md border-0 py-1.5 px-3 text-neutral-900 shadow-sm ring-1 ring-inset ring-neutral-300 placeholder:text-gray-400 focus-visible:focus-outline sm:text-sm sm:leading-6"> </div>`
-          },
-          {
-            type: 'slot',
-            name: 'header',
-            value: `<span slot='header'>Header</span>`
-          },
-          {
-            type: 'slot',
-            name: 'footer',
-            value: `<sd-button slot='footer' variant='secondary' style='width: 100%'>Footer</sd-button>`
-          },
-          {
-            type: 'attribute',
-            name: 'no-header',
-            value: false
-          }
-        ]
-      })}
-      <sd-button
-        variant="secondary"
-        @click=${() => {
-          const drawer = document.querySelector('sd-drawer');
-          if (drawer && !drawer.hasAttribute('open')) {
-            drawer.setAttribute('open', '');
-          }
-        }}
-      >
-        Open Drawer
-      </sd-button>
-    </div>`;
-  },
-  play: async ({ canvasElement }: { canvasElement?: HTMLUnknownElement }) => {
-    const el = canvasElement?.querySelector('.mouseless sd-drawer');
-    await waitUntil(() => el?.shadowRoot?.querySelector('header'));
+export const NoHeader = {
+  name: 'No Header',
+  render: () => html`
+    <div style="width: auto; height: 40vh; position: relative;">
+      <sd-drawer open no-header>
+        <p class="sd-paragraph">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie phasellus
+          dui vel id. Velit in sed
+        </p>
 
-    if (el?.shadowRoot) {
-      const header = el.shadowRoot.querySelector('header');
-      if (header) {
-        await userEvent.type(header, '{space}', { pointerEventsCheck: 0 });
-      }
-    }
-  }
+        <sd-button slot="footer">Lorem</sd-button>
+      </sd-drawer>
+    </div>
+  `
 };
```

Comment:
Please do it like in the figma file:
<img width="577" alt="image" src="https://github.com/user-attachments/assets/be462cd4-66b4-4d53-9301-2a6216aac802">

Reply:
Updated